news/2017-07-19-inclusive-science-lab.html

```diff
@@ -0,0 +1,138 @@
+                A separate panel bar for mobile displays is needed to preserve the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-smallScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+
+            <!-- This is the div that will contain the Preference Editor component -->
+            <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the desktop view.
+                A separate panel bar for desktop displays is needed to preserve the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-wideScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+        </div>
+
+        <div class="floe">
+            <header class="floe-header">
+                <h1><a rel="home" title="Home" href="../index.html" class="floe-header-logo">floe
+                    <span class="floe-header-tagline">
+                        flexible learning for open education
+                    </span></a>
+                </h1>
+            </header>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <div class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <h3>Inclusive Science Lab</h3>
+                <p class="floe-date">July 19, 2017</p>
+                <p>
+                    We've been building an integration technology called the Nexus. It allows the interconnection of arbitrary devices by co-ordinating device state over HTTP and WebSockets.
+                </p>
+                <p>
+                    Recently, we have been using it to try out designs to make science labs more inclusive. The Nexus is used to connect lab sensors, such as pH sensors and electrical conductivity sensors, to a range of different presentations and interactions. So that a student can pick the interaction that works best for them. For some students this might be a visualization and for others it might be a sonification, or a combination of different presentations.
```

Comment:
There's a funny kind of sentence fragment here: "So that a student can pick the interaction that works best for them." Perhaps "This helps a student pick the interaction that works best for them" or something like this?